database/plugin/metadata/sqlite/utxo.go

```diff
@@ -193,6 +193,27 @@ func (d *MetadataStoreSqlite) DeleteUtxosAfterSlot(
 	return nil
 }
 
+// DeleteUtxosBeforeSlot deletes Utxos marked as deleted before a given slot
+func (d *MetadataStoreSqlite) DeleteUtxosBeforeSlot(
```

Comment:
Should this be `DeleteUtxosDeletedBeforeSlot`?

Reply:
I had that originally, but we already have `DeleteUtxosAfterSlot` from a previous PR, so I made it match that.